README.md

```diff
@@ -1 +1,61 @@
-# swan-sdk
\ No newline at end of file
+markdown
+Copy code
+# SWAN SDK
+
+## Overview
+
+The SWAN SDK provides a streamlined and efficient interface for interacting with our API. It's tailored for easy management of spaces, jobs, and users, making it a versatile tool for a wide range of applications.
+
+## Features
+
+- **Easy API Integration**: Simplify your workflow with our user-friendly API client.
+- **Data Models**: Leverage our pre-built models for spaces, jobs, and users.
+- **Service Abstractions**: Gain access to high-level functionalities through our service layer.
+- **Comprehensive Documentation**: Discover detailed guides and references in the `docs/` directory.
```

Comment:
I assume the docs/ directory will come later

Reply:
I created the structure, we will add everything in steps